.vscode/launch.json

```diff
@@ -1874,7 +1876,9 @@
       "description": "Select client library to debug",
       "type": "pickString",
       "options": [
-        "product-settings"
+        "product-settings",
+        "context-registry",
+        "extension"

```

Comment:
So there should be a separate launch config for the extension client separately from these two.

It's called, ```Extension Client Tests (Integration)```. It doesn't use the Client Tests.